Trappist/src/Promact.Trappist.Web/wwwroot/app/reports/reports.routing.ts

```diff
@@ -10,7 +10,7 @@ const reportsRoutes: Routes = [
         path: 'reports',
         component: ReportsComponent,
         children: [
-            { path: '', component: TestReportComponent },
+            { path: ':id', component: TestReportComponent },
```

Comment:
I have a view report button in the View Test Page. On clicking of that button the user should be redirected to the View Report Page of that particular Test.